### PR TITLE
New Favicon

### PR DESCRIPTION
**Pull request by cindyzhang12**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

Added new favicon to the webpage!

### How to Review
- Changed code in index.html and manifest.json
- Uploaded .png and .ico versions of Iris' logo

### Related PRs
N/A

### Next steps
Wasn't super sure about the sizes of the images - I just downloaded the png from Notion and used that for every one, wasn't sure if I had to resize to the size specified in the code.

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: src/styles/Investments.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.cont {
```
Comment (dfangshuo) [inline src/styles/Investments.css]: Is there a reason this file is also in this PR? If it's not related it should not be in this PR

File: src/App.js [hunk 2]
```diff
@@ -64,6 +64,11 @@ class App extends React.Component {
     return (
       <ConnectedRouter history={history}>
         <div className="app-container">
+          <link
```
Comment (dfangshuo) [inline src/App.js]: hmm what what is this link for?

File: public/manifest.json [hunk 3]
```diff
@@ -3,17 +3,17 @@
   "name": "Create React App Sample",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "ppsc-favicon-01.ico",
```
Comment (dfangshuo) [inline public/manifest.json]: is there a reason this is ppsc-favicon-01 instead of just ppsc-favicon? If not I would recommend just naming it ppsc-favicon

Comment (cindyzhang12) [reply public/manifest.json]: Iris' file on her sprint task was ppsc-favicon-01 so I kept it that way because I thought it was named that for a reason! If not, I can change it

Comment (dfangshuo) [reply public/manifest.json]: I think that's a good enough reason - let's keep it as this!

File: src/App.js (hunk 2 above)
Comment (cindyzhang12) [reply src/App.js]: updated, this part was deleted
